pkg/virt-config/configuration.go

```diff
@@ -268,6 +268,7 @@ func defaultClusterConfig(cpuArch string) *v1.KubeVirtConfiguration {
 		LiveUpdateConfiguration: &v1.LiveUpdateConfiguration{
 			MaxHotplugRatio: DefaultMaxHotplugRatio,
 		},
+		VMRolloutStrategy: &v1.VMRolloutStrategy{Stage: &v1.RolloutStrategyStage{}},
```

Comment:
We should also add `validateConfig` validation so that in case someone sets both we get the default one.

Reply:
This is now a string